tests/test_ini.py

```diff
@@ -20,20 +31,53 @@ class TestSectionLine(unittest.TestCase):
     ]
 
     def test_invalid(self):
+        """
+        :id: b04b3e76-0c9e-4eb4-a9cd-6a1f4d527e88
+        :title: SectionLine rejects invalid lines
+        :description:
+            Verifies that lines which are not valid section headers
+            return None when parsed by SectionLine.
+        :tags: Tier 2
+        :steps:
+            1. Define a list of malformed or non-section strings.
+            2. Attempt to parse each string using SectionLine.parse().
+        :expectedresults:
+            1. Each invalid line returns None when parsed.
+            2. No exception is raised during parsing
+        """
         for l in self.invalid_lines:
             p = ini.SectionLine.parse(l)
             self.assertEqual(p, None)
 
     lines = [
         ('[section]',          ('section', None, None, -1)),
-        ('[se\ct%[ion\t]',     ('se\ct%[ion\t', None, None, -1)),
+        (r'[se\ct%[ion\t]',     (r'se\ct%[ion\t', None, None, -1)),
```

Comment:
Why did you change this line. It does not seems related to changing of doc-strings. I can see that this change removes warning message :+1: , but it should be in different commit.